### PR TITLE
119: Setup translations and translate current content

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/messages/getTranslationMessages.spec.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    it('Returns en-US translation when en-US locale is requested', () => {
+        const translation = getTranslationMessages(Locale.enUS);
+        expect(translation).toEqual(enUS);
+    });
+
+    it('Returns en-US translation when no locale is requested', () => {
+        const translation = getTranslationMessages(undefined);
+        expect(translation).toEqual(enUS);
+    });
+    it('Returns en-US translation unknown locale  is requested', () => {
```
Comment (tjeerddie) [inline packages/orchestrator-ui-components/src/messages/getTranslationMessages.spec.ts:21]:
```suggestion
    it('Returns en-US translation unknown locale is requested', () => {
```

File: apps/wfo-ui/pages/_app.tsx [hunk 2]
```diff
@@ -56,35 +59,37 @@ function CustomApp({ Component, pageProps }: AppProps) {
...
-                <OrchestratorConfigProvider
-                    initialOrchestratorConfig={initialOrchestratorConfig}
-                >
-                    <QueryClientProvider
-                        client={queryClient}
-                        contextSharing={true}
+        <TranslationsProvider>
+            <EuiProvider colorMode="light" modify={defaultOrchestratorTheme}>
+                <Head>
+                    <title>Welcome to example-orchestrator-ui!</title>
```
Comment (tjeerddie) [inline apps/wfo-ui/pages/_app.tsx:65]: use translation

Comment (DutchBen) [reply apps/wfo-ui/pages/_app.tsx:65]: The translation context is not available here yet. This requires a rearrangement of the the app page that I think is better saved for a separate PR once the translation approach is established. Or shall I just do it here right away?

File: .husky/pre-commit
Comment (ricardovdheijden) [inline .husky/pre-commit:1]: What happended with this file? I only see a size diff.

File: apps/wfo-ui/pages/_app.tsx (hunk 2 above)
Comment (ricardovdheijden) [reply apps/wfo-ui/pages/_app.tsx:65]: This _app is getting bigger and bigger with all the providers and contexts. Most of them should/might end up in the library to be consumed here. I would say to do fix this at that moment. I do agree that we eventually want to translate the title too.

File: apps/wfo-ui/translations/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+## Adding or overriding translations
```
Comment (ricardovdheijden) [inline apps/wfo-ui/translations/README.md:1]: Since the translations are part of the library as well, we could also add/move it there

**Review by ricardovdheijden — APPROVED** [review]

File: .husky/pre-commit
Comment (DutchBen) [reply .husky/pre-commit:1]: It was changed to be executable so ```chomd +x``` It wouldn't pick run the hook for me without this

**Comment by DutchBen** [comment]
Looks like this PR is messed up after a rebase. Closing and reopening

**Pull request closed without merging**